aten/src/ATen/native/native_functions.yaml

```diff
@@ -3571,15 +3571,15 @@
 # to false to make such changes explicitly illegal, in order to prevent users from
 # changing metadata of the detached tensor and expecting the original tensor to also
 # be updated.
-- func: detach(Tensor(a) self) -> Tensor(a)
+- func: detach(Tensor(a) self, bool only_backward_mode=False) -> Tensor(a)
```

Comment:
make this kwarg only?

Reply:
Ho we can do that that here? 
Will do thanks for the comment! :)